infrastructure/modules/psql/variables.tf

```
@@ -0,0 +1,315 @@
+variable "psql_SubnetCidr" {
+  type        = string
+  description = "The CIDR range to use for psql subnet"
+}
+
+variable "organization" {
+  type        = string
+  description = "The organization/service owner name"
+}
+
+variable "environment" {
+  type        = string
+  description = "The environment (ATxx/TTx/YTx)"
+}
+
+variable "location" {
+  type        = string
+  default     = "norwayeast"
+  description = "The location where the resources will be placed."
+}
+variable "psql_NetworkName" {
```

Keep to one type of formating, for terraform i prefer snake_case
```suggestion
variable "psql_network_name" {
```